website/docs/cloud-docs/api-docs/assessment-results.mdx

```diff
@@ -0,0 +1,108 @@
+---
+page_title: Assessments - API Docs - Terraform Cloud
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+
+# Assessment Results API
+
+An Assessment Result is the record of an instance of continuous validation such as [Drift Detection](/cloud-docs/workspaces/settings/drift-detection).
+
+## Show assessment result
+
+`GET api/v2/assessment-results/:assessment_result_id`
+
+| Parameter               | Description              |
+| ----------------------- | ------------------------ |
+| `:assessment_result_id` | The assessment result ID |
+
+
+### Sample Request
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  https://app.terraform.io/api/v2/assessment-results/asmtres-cHh5777xm
+```
+
+### Sample Response
+
+```json
+{
+  "id": "asmtres-UG5rE9L1373hMYMA",
+  "type": "assessment-results",
+  "data": {
+    "attributes": {
+      "drifted": true,
+      "succeeded": true,
+      "error-msg": null,
+      "created-at": "2022-07-02T22:29:58+00:00",
+    },
+    "links": {
+      "self": "/api/v2/assessment-results/asmtres-UG5rE9L1373hMYMA/"
+      "json-output": "/api/v2/assessment-results/asmtres-UG5rE9L1373hMYMA/json-output"
+      "json-schema": "/api/v2/assessment-results/asmtres-UG5rE9L1373hMYMA/json-schema"
+      "log-output": "/api/v2/assessment-results/asmtres-UG5rE9L1373hMYMA/log-output"
+    }
+  }
+}
+```
+
+## Retrieve the JSON output from the assessment execution
+
+The following endpoints retrieve files documenting the plan, schema, and logged runtime associated with the specificied assessment result.
+The reponses do not adhrere to JSON API spec and are intended to provide complete context for the assessment result's otherwise summary level of information.
+
+`GET api/v2/assessment-results/:assessment_result_id/json-output`
+
+This endpoint returns the JSON plan output associated with the assessment result.
```

Comment:
I think we typically do include an explanation underneath, but here it's a bit confusing because there are three in a row and my brain had to work harder than I wanted it to in order to match this description with the thing ABOVE it and not the thing below it. Could we try moving them to come before the endpoints and say "The following endpoint..."

Reply:
I should probably just split these out more too.